### PR TITLE
Refresh /hpc and add contact modal

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Refresh /hpc and add contact modal
- Based on [copydoc](https://docs.google.com/document/d/1d4E1IA2mEu72q277sihNSkl7QRKsU1ezwA3jVgJ0TCE/edit#) and [design](https://www.figma.com/proto/NgiuI3rQmyLBljKHyJrZXK/HPC?page-id=7%3A3918&node-id=7%3A3919&viewport=639%2C754%2C0.33&scaling=min-zoom)

## QA

- Visit https://ubuntu-com-12231.demos.haus/hpc
- Compare against copydoc and design

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12111

### CODE REVIEW

**Comment by petesfrench** [comment]
@elioqoshi A couple things I wasn't sure about:
1. The design finishes at the webinar/case study section, so can the rest be left how it is? There are two sections that clash quite heavily.
image
2. In the copydoc the section for 'Ceph' is along with the different Canonical products, but I suppose should be with the other partners?

**Comment by elioqoshi** [comment]
The latest designs are here and they should have the right latest content: 
https://www.figma.com/file/NgiuI3rQmyLBljKHyJrZXK/HPC?node-id=7%3A3918

**Comment by petesfrench** [comment]
@elioqoshi I have updated the page according to the design, can you take a look please?

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @petesfrench! Just a few things:

File: templates/hpc/index.html [hunk 1]
```diff
@@ -228,448 +213,390 @@ <h2 class="p-heading--5">
...
+
+  <section class="p-strip u-no-padding--bottom">
+    <div class="u-fixed-width">
+      <hr>
     </div>
-  </section>
-  <div class="u-fixed-width">
-    <hr class="p-seperator" />
-  </div>
-  <section class="p-strip">
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/hpc/index.html:477]: Good catch! It has moved to the section above

**Comment by elioqoshi** [comment]
This is looking good to me! Maybe except the logo alignments, which seems different from what @lyubomir-popov suggests?

**Comment by lyubomir-popov** [comment]
I agree, the logos are far too large.

**Comment by petesfrench** [comment]
@elioqoshi @lyubomir-popov What do you suggest I do here? This uses the standard vanilla pattern and logos from the updated logo collection Lyubo put together.

**Comment by petesfrench** [comment]
@lyubomir-popov Ok I understand, I thought were were talking about the logo-section at the top of the page. Updated.

File: templates/hpc/index.html [hunk 2]
```diff
@@ -120,26 +109,15 @@ <h1>Your trusted partner for HPC and supercomputing</h1>
...
-            width="80",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-        </div>
-        <hr class="u-sv1" />
         <h2>HPC is used across multiple sectors and industries</h2>
         <p>
           From energy to healthcare, we help industry leaders ease operations and break new ground with the power of compute.
```
Comment (mtruj013) [inline templates/hpc/index.html:123]: "Accelerate discovery in:" is bold text on the design

File: templates/hpc/index.html [hunk 3]
```diff
@@ -157,66 +135,73 @@ <h2>HPC is used across multiple sectors and industries</h2>
...
+          Cluster management at scale
+        </h3>
+      </div>
+      <div class="col-6">
         <p>
-          The ultimate experience for bare metal resource discovery, management, delivery and observability.
+          <strong>MAAS: super fast cluster deployment and delivery</strong> <br> The ultimate experience for bare metal resource discovery, management, delivery and observability.
         </p>
         <p>
           Deploy HPC and supercomputing clusters with ease.
```
Comment (mtruj013) [inline templates/hpc/index.html:207]: Gloria [requested](https://docs.google.com/document/d/1d4E1IA2mEu72q277sihNSkl7QRKsU1ezwA3jVgJ0TCE/edit?disco=AAAAeQXgTmk) this be made a single paragraph

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench, looks great! Left a few more comments, but all for super small stuff.

**Comment by mtruj013** [comment]
Oh also, forgot to mention, the Mercedes logo is different from the one the Pdm requested, not sure if it matters or not though!

File: templates/hpc/index.html [hunk 4]
```diff
@@ -157,66 +135,73 @@ <h2>HPC is used across multiple sectors and industries</h2>
...
           <li class="p-list__item is-ticked">Media and entertainment</li>
           <li class="p-list__item is-ticked">Research and universities</li>
           <li class="p-list__item is-ticked">Weather</li>
         </ul>
-        <p>
-          <a class="p-button--positive" href="/hpc/use-cases">Use cases</a>
-        </p>
       </div>
+      <p>
+        <a class="p-button--positive" href="/hpc/use-cases">See use cases</a>
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/hpc/index.html:168]: We should try and give links and buttons directive text. I spoke with @al-bakalova and she has updated doc accordingly

**Comment by petesfrench** [comment]
@lyubomir-popov Could you take another look at this please

**Comment by sowasred2012** [comment]
@lyubomir-popov could either of you give a design review on this please?

**Comment by elioqoshi** [comment]
This looks good to me now, thank you!

**Pull request merged**